src/index.ts

```diff
@@ -2,7 +2,7 @@ import type { AuthForHAR } from './lib/configure-security';
 import type { Extensions } from '@readme/oas-extensions';
 import type { PostDataParams, Request } from 'har-format';
 import type Oas from 'oas';
-import type { SchemaWrapper } from 'oas/dist/operation/get-parameters-as-json-schema';
+import type { SchemaWrapper } from 'oas/operation/get-parameters-as-json-schema';
```

Comment:
very cool that we no longer have to pull these out of the dist.

Reply:
With the new exports key in package.json, you have now a nice public api :)

Reply:
unclear, but will look into it after i get this published